<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ Trolltech AS. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>AppearancePage</class>
 <widget class="QWidget" name="AppearancePage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>&amp;Style</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="cmbStyle" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>&amp;Tune Palette</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Quick</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="StyledButton" native="1" name="pbBuild" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Choose a color</string>
        </property>
        <property name="whatsThis" >
         <string>Choose the effect-color for the generated palette.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="rbCompute" >
        <property name="text" >
         <string>Compute Details</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QRadioButton" name="rbDetails" >
        <property name="text" >
         <string>Show Details</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4" >
       <widget class="QTreeView" name="paletteView" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreview" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Previe&amp;w</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="rbDisabled" >
        <property name="text" >
         <string>Disabled</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="rbInactive" >
        <property name="text" >
         <string>Inactive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="rbActive" >
        <property name="text" >
         <string>Active</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="PreviewFrame" native="1" name="previewFrame" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledButton</class>
   <extends>QWidget</extends>
   <header>styledbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PreviewFrame</class>
   <extends>QWidget</extends>
   <header>preview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbStyle</tabstop>
  <tabstop>pbBuild</tabstop>
  <tabstop>rbCompute</tabstop>
  <tabstop>rbDetails</tabstop>
  <tabstop>paletteView</tabstop>
  <tabstop>rbActive</tabstop>
  <tabstop>rbInactive</tabstop>
  <tabstop>rbDisabled</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
